pkg/devfile/adapters/docker/utils/utils.go

```diff
@@ -66,3 +91,80 @@ func GetProjectVolumeLabels(componentName string) map[string]string {
 	}
 	return volumeLabels
 }
+
+// GetSupervisordVolumeLabels returns the label selectors used to retrieve the supervisord volume
+func GetSupervisordVolumeLabels() map[string]string {
+	image := adaptersCommon.GetBootstrapperImage()
+	_, _, _, imageTag := util.ParseComponentImageName(image)
+
+	supervisordLabels := map[string]string{
+		"name":    adaptersCommon.SupervisordVolumeName,
+		"type":    "supervisord",
+		"version": imageTag,
+	}
+	return supervisordLabels
+}
+
+// CreateAndInitSupervisordVolume creates the supervisord volume and initializes it with odo init
```

Comment:
What do you mean by `initializes it with odo init`?

Reply:
updated to say supervisord bootstrap, i meant odo init repo earlier